### ISSUE TITLE
RoVer is pulling old user names

### ISSUE THREAD

**Issue by DebraG13** [evaera/RoVer#73]
When members that are already verified type, RoVer is changing their names to previous Roblox usernames. If the force update command is used, it will pull the correct name again.

**Comment by evaera** [comment]
@DebraG13 This is not a bug. This is the intended behavior. In order to check what someone's current username is, we have to issue a request to the Roblox API. It is not feasible to do this every time someone sends a message because it would use a large amount of server resources (both RoVer's server and Roblox's servers) for data that rarely changes. 

Therefore, the usernames are cached unless we know the user is specifically trying to trigger a manual update (such as with the `!verify` command or `!update` command).  Users can use this command at any time to update their name if they change it. Additionally, the cache is cleared at set intervals, which on the hosted version is every two hours. So even if the user does not run any commands, their username will eventually be correct. It's just not instant.

**Comment by DebraG13** [comment]
See, that's what I thought also. But a couple in our server, that have been there a while and verified with us at the start, all of a sudden when they typed we got a hit in our admin channel saying it was changed. And it changed to an old username. They were verified in our server for a while, and had been typing all day, then bam, it happened. Not the first time this randomness has happened either. Once we do !update <user> it will go back to the right name.
